packages/cypress-commands/src/setups/enableNetworkShim/index.js

```diff
@@ -38,7 +39,7 @@ export function enableNetworkShim({
         })
     })
 
-    after(() => {
+    afterEach(() => {
```

Comment:
Won't this change cause the file to be written after each individual test?  Is that a problem?

Reply:
My understanding is that this will overwrite the file with a bit more information after every test, which is probably OK, but could cause performance or even race condition problems when files are large...?

Reply:
Yeah very true. However, since the cypress [clears aliases between tests](https://github.com/cypress-io/cypress/issues/665#issuecomment-330600042), we really do have to stick to the `beforeEach` and `afterEach` hook. This approach is far from ideal, but luckily the cypress-plugin based approach which we'll introduce soon solves this issue.

I've already tested this current implementation in the data-capture app and found the following:
- No race conditions with reading/writing files occurred (could have been a stroke of luck, but it was quite a substantial test-suite)
- I compared the cumulative/total run-time of a capture-run with a normal run and found that the capture run was 17% slower. Quite bad, but not terrible.

My recommendation for now would be to merge the current PR so we have a fix for v5, but to also take this as a cue to press ahead with the plugin based version, which will be in v6.

Reply:
FYI, this will be the implementation when we switch to the cypress-plugin based approach. This should be much less problematic because the state object is created by the plugin when the run starts and files are written when the run ends, during the run it's simply getting and setting this state:
https://github.com/dhis2/cli-utils-cypress/blob/b8b67b2affaf114a4777d4d90f79dc62a048e5b8/packages/cypress-commands/src/setups/enableNetworkShim/index.js#L6-L41

Reply:
Great, thanks for the analysis!  I agree, let's get this in